kevm-pyk/pyproject.toml

```diff
@@ -13,7 +13,8 @@ authors = [
 [tool.poetry.dependencies]
 python = "^3.10"
 pathos = "*"
-pyk = { git = "https://github.com/runtimeverification/pyk.git", tag="v0.1.343" }
+pyk = { git = "https://github.com/runtimeverification/pyk.git", tag="v0.1.350" }
+# pyk = { path = "/home/ifrostizz/rv/pyk/", develop=true }
```

Comment:
@iFrostizz please remove this comment. Also make sure to do a self-review on the GitHub web interface before asking for review from others. See our GitHub guidelines: https://runtimeverification.com/blog/github-guidelines-for-collaborative-coding